core/controllers/library.py

```diff
@@ -143,6 +156,87 @@ def get(self):
         self.render_json(self.values)
 
 
+class LibraryGroupPage(base.BaseHandler):
+    """The page for displaying top rated and recently published explorations
+    """
+
+    def get(self, rank_method):
+        """Handles GET requests."""
+
+        search_mode = 'search' in self.request.url
+        rank_mode = 'explorations' in self.request.url
```

Comment:
Why needed? It's not read by the frontend.

That said, in this page, it's obviously True...